modules/shared/components/molecules/Footer/Footer.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import type { FC, ReactElement } from 'react';
+import type { IFooter } from './IFooter';
```

Comment:
better to name it `postFooter` as we most probably will have a footer for the whole app